GuildedRoseLLC/Items/RemoteItemRepository.swift

```diff
@@ -0,0 +1,32 @@
+import UIKit
+
+public class RemoteItemRepository: ItemRepository {
+    
+    public init() {}
+    
+    public func getItems(onSuccess: @escaping ([Item]) -> Void) {
```

Comment:
Right now, this class is entirely untested, which is/was expected, since that's what we talked about.

That said, now that this class exists, I think it would make sense to take a closer look and see how this code could be refactored to minimize the amount of untested code.

I think this would be a great topic for a future whiteboarding session with one or more mentors.


To be explicit: A single, large, untested class if fine for this pull request.